about.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+
+    <header>
+        <nav class="">
+            <div class="pre-header">
+                <div class="pre-header-flex">
+                    <ion-icon name="logo-facebook" class="icon"></ion-icon>
+                    <ion-icon name="logo-twitter" class="icon"></ion-icon>
+                    <span class="tabs">English</span>
+                    <span class="tabs">Language</span>
+                    <span class="tabs">Logout</span>
+                </div>
+            </div>
+            <div class=" navbar-expand-md desktop-menu">
+                <a class="navbar-brand" href="index.html"><img class="nav-logo" src="img/Kenya-Experience.svg" alt="logo"></a>
+                <ul class="navbar-nav">
+                    <li class="nav-item"><a class="nav-link" href="about.html">About</a></li>
+                    <li class="nav-item"><a class="nav-link" href="">Program</a></li>
+                    <li class="nav-item"><a class="nav-link" href="">Join</a></li>
+                    <li class="nav-item"><a class="nav-link" href="">Sponsor</a></li>
+                    <li class="nav-item"><a class="nav-link" href="">News</a></li>
+                    <li class="nav-item"><a class="butt pt-0 pb-0" href="">KenyaTravel</a></li>
```

Comment:
- [ ] [ **OPTIONAL** ] Good job adding the missing part in your header as requested by my fellow CR @zieeco , however I noticed that the nav links are not responsive , when the user click on a nav-link , it should bring him to a corresponding section 
 You could do `<li class="nav-item"><a class="nav-link" href="#sponsor-id">Sponsor</a></li>` 
Add an `section id` as `href`

![Screenshot from 2023-02-09 17-32-23](https://user-images.githubusercontent.com/80458199/217859347-ff9040bc-4220-4fdf-8ba6-88f2259514df.png)